packages/client-core/src/common/services/MediaInstanceConnectionService.ts

```diff
@@ -36,19 +36,15 @@ type InstanceState = {
 export const MediaInstanceState = defineState({
   name: 'MediaInstanceState',
   initial: () => ({
-    instances: {} as { [id: string]: InstanceState },
+    instances: {} as { [id: string]: InstanceState | undefined },
```

Comment:
Having this typed as undefined would mean there is potential for empty keys to be stored here. We do not want this.